db/migrate/20220620054530_add_temp_new_email_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddTempNewEmailToUsers < ActiveRecord::Migration[6.1]
+  def change
+    add_column :users, :temp_new_email, :string
```

Comment:
1. Not a fan of the column name. Why not just call this `new_email`? 😅 
2. Shouldn't this be a `citext` to match the `email` column?